oauth2_http/java/com/google/auth/oauth2/ImpersonatedCredentials.java

```diff
@@ -425,6 +595,15 @@ public HttpTransportFactory getHttpTransportFactory() {
       return transportFactory;
     }
 
+    public Builder setQuotaProjectId(String quotaProjectId) {
+      this.quotaProjectId = quotaProjectId;
+      return this;
+    }
+
+    public String getQuotaProjectId() {
```

Comment:
This method is not tested. Is it needed at all? It can probably be removed at no loss.

Reply:
Thanks for pointing it out! It is not used and I removed it.